tests/roundtrips.py

```diff
@@ -28,6 +28,9 @@
     "clear": ["cpp", "py", "rust"],  # Don't need it, API example roundtrips cover it all
     "mesh3d": ["cpp", "py", "rust"],  # Don't need it, API example roundtrips cover it all
     "time_series_scalar": ["cpp", "py", "rust"],  # Don't need it, API example roundtrips cover it all
+    "scalar": ["cpp", "py", "rust"],  # TODO(jleibs)
+    "series_line": ["cpp", "py", "rust"],  # TODO(jleibs)
+    "series_point": ["cpp", "py", "rust"],  # TODO(jleibs)
```

Comment:
do we _really_ need roundtrip tests for these? worth bringing up as a post-standup topic how many of these we want/need in the future

Reply:
yeah, TODO there until I've at least made sure they are covered somehow. With it's implicit via code-examples or explicit via RTT.